source/deploy/mattermost-changelog.md

```diff
@@ -7,10 +7,86 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
 
 - See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) documentation for details on upgrading.
 - See the [changelog in progress](https://bit.ly/2nK3cVf) for details about the upcoming release.
-- **Self-Hosted Releases Prior to v9.2**: See the [Mattermost Legacy Self-Hosted Changelog](https://docs.mattermost.com/deploy/legacy-self-hosted-changelog) for details.
-- **Cloud Releases Prior to v9.2**: See the [Mattermost Legacy Cloud Changelog](https://docs.mattermost.com/deploy/legacy-cloud-changelog) for details.
+- **Self-Hosted Releases Prior to v9.2**: See the [Mattermost Legacy Self-Hosted Changelog](https://docs.mattermost.com/deploy/legacy-self-hosted-changelog.html) for details.
+- **Cloud Releases Prior to v9.2**: See the [Mattermost Legacy Cloud Changelog](https://docs.mattermost.com/deploy/legacy-cloud-changelog.html) for details.
 ```
 
+## Release v9.7 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: April 16, 2024**
+
+```{Important}
+If you upgrade from a release earlier than v9.6, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Added Mattermost [AI plugin](https://github.com/mattermost/mattermost-plugin-ai) to pre-packaged plugins.
+ - Pre-packaged Calls version [v0.25.1](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v0.25.1).
+ - Pre-packaged GitHub plugin version [v2.2.0](https://github.com/mattermost/mattermost-plugin-github/releases/tag/v2.2.0).
+ - Added a user limit enforcement.
+ - Upgraded ``@mattermost/client`` and ``@mattermost/types`` to support TypeScript v5.x.
+ - The first emoji is now auto-selected in the emoji picker.
+ - Added markdown support for batched email notifications.
+ - Users’ timezone is now used in batched email notifications.
+ - Removed a conflicting class (``help-text``) from the interactive dialog field description to resolve the black text color in the dark theme.
+ - Updated the user interface of **Team Settings** modal.
+ - Promoted Simplified Chinese to Beta, and downgraded Hungarian and Spanish languages to Beta.
+ - Improved the opening animation of the user settings modal.
+
+#### Administration
```

Comment:
@amyblais can we include the change for the safety limits that we introduced with https://github.com/mattermost/mattermost/pull/26511 cc// @jasonblais 

```suggestion
#### Administration

- Enforced safety limit in compiled Team Edition and Enterprise Edition deployments when enterprise scale and access control automation features are unavailable and count of users who are registered and not deactivated exceeds 11,000. ERROR_SAFE_LIMITS_EXCEEDED.
```